BrickKit/bricks/src/androidTest/res/layout/text_brick_vm.xml

```diff
@@ -20,7 +20,7 @@
             android:textColor="@android:color/black"
             android:textSize="31sp"
             android:background="@android:color/white"
-            android:padding="@dimen/default_brick_content_padding"
```

Comment:
should this be eight_dp?

Reply:
didn't feel worth it to add a resource for a test only layout